functions/Install-DbaFirstResponderKit.ps1

```diff
@@ -113,32 +117,47 @@ Logs into server1\instance1 with SQL authentication and then installs the FRK in
 			Stop-Function -Message "Couldn't download the First Responder Kit. Download and install manually from https://github.com/BrentOzarULTD/SQL-Server-First-Responder-Kit/archive/master.zip." -ErrorRecord $_
 			return
 		}
-		
 	}
 	
-	PROCESS {
-		Foreach ($instance in $SqlInstance) {
+	process {
+		if (Test-FunctionInterrupt) { return }
+		
+		foreach ($instance in $SqlInstance) {
 			try {
-				Write-Host "Connecting to $instance"
-				$Connection = Connect-DbaSqlServer -SqlInstance $instance -Credential $sqlcredential
+				Write-Message -Level Verbose -Message "Connecting to $instance"
+				$server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $sqlcredential
 			}
 			catch {
-				Write-Host "Failed to connect to $instance : $($_.Exception.Message)"
+				Stop-Function -Message "Failed to connect to: $instance" -Target $instance -ErrorRecord $_ -Continue
 			}
 			
+			Write-Message -Level Output -Message "Starting installing/updating the First Responder Kit stored procedures in $database on $instance"
+			
 			# Install/Update each FRK stored procedure
-			Get-ChildItem $zipfolder -Filter sp_Blitz*.sql | Foreach-Object {
-				if ($_.Name -ne "sp_BlitzRS.sql") {
-					Write-Host "Installing/Updating $_."
+			foreach ($script in (Get-ChildItem $zipfolder -Filter sp_Blitz*.sql)) {
+				$scriptname = $script.Name
+				if ($scriptname -ne "sp_BlitzRS.sql") {
+					Write-Message -Level Output -Message "Installing/Updating $scriptname"
+					$sql = [IO.File]::ReadAllText($script.FullName)
 					
-					$sql = [IO.File]::ReadAllText($_.FullName)
+					if ($scriptname -eq "sp_BlitzQueryStore.sql") {
+						if ($server.VersionMajor -lt 13) { continue }
+					}
 					
-					$null = $Connection.databases[$Database].ExecuteNonQuery($sql)
+					foreach ($query in ($sql -Split "\nGO\b")) {
```

Comment:
GO is an SSMS keyword and SMO's Execute chokes on it so a lot of times, online scripts will have to be split at GO. `\n` means new line and `\b` is "boundary". I'm not awesome with regex but this seemed to work 💯